tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+envlist = coverage-clean, py27, py33, py34, py35, coverage-report
+
+[testenv]
+commands =
+    coverage run setup.py ftest
+setenv =
+  COVERAGE_FILE=.coverage.{envname}
+deps =
+    {toxinidir}/zope.app.publication-4.0.0a1.dev0.tar.gz
+    coverage
+    zope.testrunner
+    ; ZEO pin to get compatibility for Python 3.3
+    ZEO <5.0
```

Comment:
What's this about?  The latest ZEO on PyPI is 4.3.0.  Are there plans to drop Python 3.3 support in ZEO 5.0 and require 3.4+?